src/Uno.UI/UI/Xaml/IDependencyObjectParse.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Windows.UI.Xaml
```

Comment:
I wonder if we should not expose that interface, and use a helper method that will set that property. This will be easier to migrate in the future if we decide to remove that dependency.

Reply:
I tried to use helper methods in other cases for exactly that reason; in this case though, we want to set `IsParsing=true` before any other properties are set in the object initializer. It's not really possible to use a helper method without completely re-architecting the way we create objects from Xaml.